PerformancePkg/Application/FbptDump/FbptDump.c

```diff
@@ -0,0 +1,290 @@
+/** @file FbptDump.c
+ This user-facing application dumps FBPT into a file.
+
+Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>

```

Comment:
I noticed `BaseMemoryLib` and `MemoryAllocationLib` are not in the INF library classes section. Can you please double check library usage?